schema/types/QuantitySourceType.schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/QuantitySourceType.schema.json",
+  "title": "Enum - Quantity Source Type",
+  "description": "Enumeration of quantity source types describing where a quantity value came from",
+  "type": "string",
+  "enum": [
+    "USER_ESTIMATED",
```

Comment:
Not sure I like this naming (yes I know I proposed it) because I don't know that it's clear who the "USER" is in different processing contexts.

Reply:
Updated in latest commit. Thx for the suggestion.

Reply:
Updated and ready for review.